Danki2/Assets/Scripts/Abilities/InstantCast/DaggerThrow.cs

```diff
@@ -22,6 +22,11 @@ public void OnCollision(GameObject gameObject)
     {
         if (gameObject.TryGetComponent<Actor>(out var actor))
```

Comment:
We can avoid doing `TryGetComponent` now, we can just check if the game object has a relevant tag (see comment below about having TagUtils) and if it does just do `GetComponent`. Same for the Fireball.